integration_tests/src/tests/schedule_update.rs

```diff
@@ -124,8 +124,9 @@ pub fn schedule_update() -> Result<()> {
     compare_video_dumps(
         OUTPUT_DUMP_FILE,
         &new_output_dump,
-        &[Duration::from_millis(500), Duration::from_micros(3500)],
+        &[Duration::from_millis(500)..Duration::from_millis(3500)],
         20.0,
+        0.0,
```

Comment:
- pack all optional arguments into struct that implements default
- change argument for allowed invalid frames to define number of frames